DigitalLearningSolutions.Web/Controllers/Register/RegisterAtNewCentreController.cs

```diff
@@ -102,13 +102,11 @@ public IActionResult PersonalInformation(InternalPersonalInformationViewModel mo
             {
                 var delegateAccount = userService.GetUserById(User.GetUserIdKnownNotNull())!.DelegateAccounts
                     .SingleOrDefault(da => da.CentreId == model.Centre);

```

Comment:
This can be replaced with
```
var delegateAccount = userService.GetUserById(User.GetUserIdKnownNotNull())!.GetCentreAccountSet(model.Centre.Value)?.DelegateAccount;
```
if we would like to use `GetCentreAccountSet` in more (all?) appropriate places

Reply:
Might as well. I've changed to use it here.